test/util/nightwatch-commands/axeCheck.js

```diff
@@ -19,16 +19,16 @@ export function command(context, config, _callback) {
   }, [axeSource]);
 
   // Run axe checks and report
-  this.executeAsync((innerContext, done) => {
+  this.executeAsync((innerContext, rules, done) => {
```

Comment:
OK, updated so now you can use
```
.axeCheck('.main', { rules: ['section508', 'wcag2a'] })
```
or any other rules from https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter to achieve extra awesome accessibility